templates/Makefile.build-harness

```diff
@@ -136,13 +137,19 @@ build-harness/shell-slim builder-slim: ARGS := $(if $(TARGETS),$(TARGETS),-l ||
 build-harness/shell-slim builder-slim: ENTRYPOINT := $(if $(TARGETS),/usr/bin/make,/bin/bash)
 build-harness/shell-slim builder-slim: build-harness/runner
 
-pr/auto-format pr/readme pr/pre-commit pr/github-update tf14-upgrade : ENTRYPOINT := /usr/bin/make
+precommit/terraform pr/auto-format pr/readme pr/pre-commit pr/github-update tf14-upgrade : ENTRYPOINT := /usr/bin/make
```

Comment:
The target from the original PR was renamed to `terraform/precommit`. You can call it here , or deprecate . I also noticed that you don't actually call tflint here.

Reply:
> The target from the original PR was renamed to `terraform/precommit`. You can call it here , or deprecate . I also noticed that you don't actually call tflint here.

We do call `terraform/precommit` here: https://github.com/cloudposse/build-harness/blob/6027a1d5567d8f45615dfcc5f562658d4db09fdf/templates/Makefile.build-harness#L148

which calls `terraform/tflint` here: https://github.com/cloudposse/build-harness/blob/6027a1d5567d8f45615dfcc5f562658d4db09fdf/modules/terraform/Makefile#L128